Shared/Bytes.swift

```diff
@@ -11,14 +11,12 @@ public typealias GUID = String
  */
 open class Bytes {
     open class func generateRandomBytes(_ len: UInt) -> Data {
-        let len = Int(len)
-        var data = Data(count: len)
-        data.withUnsafeMutableBytes { (p: UnsafeMutablePointer<UInt8>) in
-            if SecRandomCopyBytes(kSecRandomDefault, len, p) != errSecSuccess {
```

Comment:
Your new code definitely works the same and the outcome is the same (a cryptographically strong random array of bytes).

However, to fix this warning we don't actually have to change the functions being called. We can use the `baseAddress` instead which is what is recommended:

IE:

```
data.withUnsafeMutableBytes {
	if let baseAddress = $0.baseAddress, SecRandomCopyBytes(kSecRandomDefault, len, baseAddress) != errSecSuccess {
		fatalError("Random byte generation failed.")
	}
}
```

Which is the exact same minus the deprecation warning :)